x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.ts

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
Also ~90% lifted from https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts. Will point out differences in comments following.